rust/ql/test/query-tests/security/CWE-312/CONSISTENCY/ExtractionConsistency.expected

```diff
@@ -0,0 +1,3 @@
+extractionWarning
+| test_logging.rs:90:12:90:30 | expected R_PAREN |
+| test_logging.rs:90:12:90:30 | macro expansion failed: the macro '$crate::__private_api::format_args' expands to ERROR but a Expr was expected |
```

Comment:
Looks like there's an error on line 90. Maybe it's the `:?`? I'm not familiar with the `error!` macro.

Reply:
Ah, I had assumed this must be an extractor bug, but I never actually ran `rustc` on this code...

Reply:
I've made the test runnable, and ran it.  The syntax on line 90 is accepted by the compiler and produces the expected output (well, I think `SimpleLogger` is discarding the keys / values / structured part, presumably a more sophisticated logger would use it).

Reply:
I see. Maybe it's because the key-value syntax requires [the `kv` feature](https://docs.rs/log/latest/log/kv/index.html) to be turned on, but the extractor always uses the default features.

Reply:
Sounds like there's nothing wrong with the test then (it does turn the kv feature on).  This is probably explains why I couldn't get the results on lines 77 and 78 (which use the kv feature).

Reply:
Yes, I think you're right :)